src/containers/WithQAing.jsx

```diff
@@ -1,10 +1,9 @@
 /* eslint-disable react/jsx-props-no-spreading */
 import React from 'reactn';
 import PropTypes from 'prop-types';
-import includes from 'lodash/includes';
-import isObject from 'lodash/isObject';
 import filter from 'lodash/filter';
 import flattenDeep from 'lodash/flattenDeep';
+import { answerAccessorGets } from '../lib/answers';
```

Comment:
started moving these formatter utilities out of `WithQAing`, beginning with the ones needed to parse answers for expanding universe and surveying the solar system